.github/workflows/prod-release.yaml

```diff
@@ -93,6 +93,19 @@ jobs:
           docker tag ${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller:${{ github.event.inputs.tag }}-linux_arm64 ${{ secrets.PROD_MXP_AWS_ACCOUNT }}.dkr.ecr.eu-south-1.amazonaws.com/amazon/appmesh-controller:${{ github.event.inputs.tag }}-linux_arm64
           docker push ${{ secrets.PROD_MXP_AWS_ACCOUNT }}.dkr.ecr.eu-south-1.amazonaws.com/amazon/appmesh-controller:${{ github.event.inputs.tag }}
           docker push ${{ secrets.PROD_MXP_AWS_ACCOUNT }}.dkr.ecr.eu-south-1.amazonaws.com/amazon/appmesh-controller:${{ github.event.inputs.tag }}-linux_arm64
+      - name: Configure AWS Credentials (prod-CGK)
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-west-2
```

Comment:
this is slightly confusing that the region here is `us-west-2` but this is how we're doing it for all the other regions. @jeremymill i assume we do this because this is part of the initial tag we generate when we build the image?

Reply:
The story behind this is that we had problems with the STS endpoint for af-south-1, but creds are not region specific, so we just get creds from us-west-2 regardless of which region we actually need to use.

Reply:
Of course, china regions and any other aws partition would need to be in their respective region.

Reply:
gotcha thx for the context